test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4247,45 +4250,97 @@ def _load_configuration(self, infile, objtype=None):
+                # Load hooks
+                elif k == "hooks":
+                    for hpath in conf['hooks']:
+                        h_name = hpath.split(os.sep)[-1]
+                        hook_name = h_name[:-3]
+                        hook_name = str(hook_name)
+                        if hook_name.startswith("PBS_"):
+                            print("PBS hook = %s" % hook_name)
+                        else:
+                            print("Site hook = %s" % hook_name)
+                            cf_hpath = ""
+                            if ".PY" in hpath:
+                                cf_hpath = hpath.replace('.PY', '.CF', 1)
+                                hk_hpath = hpath.replace('.PY', '.HK', 1)
+                                cmd = qmgr + " -c 'create hook %s'" % hook_name
+                                self.du.run_cmd(
+                                    self.hostname, cmd, sudo=True,
+                                    as_script=True)
+                                hk_contents = base64.decodestring(
+                                    conf['hooks'][hk_hpath])
+                                hk_vals = self.utils.convert_to_dictlist(
+                                    hk_contents.splitlines())
+                                for k, val in hk_vals[0].items():
+                                    if k == 'event':
+                                        e_v = val.split(',')
+                                        for v in e_v:
+                                            cmd = qmgr + \
+                                                " -c 's h %s event+=%s'" % (
+                                                    hook_name, v)
+                                            self.du.run_cmd(
+                                                self.hostname, cmd,
+                                                sudo=True, as_script=True)
+                                    cmd = (qmgr +
+                                           " -c 's h %s %s=%s" % (hook_name,
+                                                                  k, val))
+                                hook_py_b64str = conf['hooks'][hpath]
+                                hook_py_str = base64.decodestring(
+                                    hook_py_b64str)
+                                fn = self.du.create_temp_file()
+                                with open(fn, 'w') as fd:
+                                    fd.write(hook_py_str)
+                                i_h = " -c 'i h %s application/x-python"
+                                i_h += " default %s'"
+                                import_py_cmd = (qmgr + "%s" % i_h
+                                                 % (hook_name, fn))
+                                self.du.run_cmd(
+                                    self.hostname, import_py_cmd,
+                                    sudo=True, as_script=True)
+                                # os.unlink(fn)
```

Comment:
Remove this line if not intended to use.

Reply:
Similar set of comments at line 4261. Have a look at them and remove it.